### PR TITLE
Docker container and test setup for EL8

### PR DESCRIPTION
**Pull request by vitodb**
This PR provides Dockerfiles, entrypoint scripts,
Jenkinsfile wf configuration, github actions and alike for EL8.
Updates to the Jenkins project are required as well.
Where needed previous EL7 files have been migrated to an EL7 folder.
Currently github actions for EL8 are skipped.
This also includes an update to setup.py to add distribution name to the RPM.
This PR goes in tandem with [PR 430 in decisionengine_modules](https://github.com/HEPCloud/decisionengine_modules/pull/430).

### CODE REVIEW

**Review by mambelli — CHANGES_REQUESTED** [review]
I think leaving EL version towards the end of the director tree is better. Some packaging methods (e.g. rpms) may not even need 2 directories:
```
package/EL8/ci/Dockerfile -> package/ci/el8/Dockerfile
```

File: setup.py [hunk 1]
```diff
@@ -54,6 +54,7 @@
     "coverage == 6.1.2",  # https://github.com/nedbat/coveragepy/issues/1300
     "pytest >= 6.2.2, < 7.0",  # pytest 7 incompatible with pytest-postgres < 4
     "pytest-cov >= 2.11.1",
+    "flake8 < 5.0.0",  # flake8 5 incompatible with pytest-flake8 > 1.0.7
```
Comment (mambelli) [inline setup.py:57]: This maybe should be (was actually) in a different PR

**Review by mambelli — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (vitodb) [reply setup.py]: this has been addressed

File: package/EL8/container/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<!--
```
[the remark this answers is not in the document]
Comment (vitodb) [reply package/EL8/container/README.md]: this has been addressed

**Review by mambelli — APPROVED** [review]

**Pull request merged**